Documentation/cilium_operator.rst

```diff
@@ -0,0 +1,212 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+Cilium Operator
+===============
+
+.. note:: This document provides a technical overview of the Cilium
+          operator and describes the cluster-wide operations it is
+          responsible for.
```

Comment:
Nit, does this need to be a note? Seems like this is just the summary text you'd expect at the top of the document. As nice as the callout boxes make the docs look, overusing them too much will encourage users to ignore them when there's really something important we want to call out.

Reply:
Yeah, that makes sense actually. Thanks for pointing this out, I have updated the pull request to not have this as a note.